schema/software/template_parameters.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$id": "software/template_parameters.json",
```

Comment:
we should also have an example of this for the tests

Reply:
test added, and modified key fields to be array of strings instead of objects